packages/registry/src/ports/PoapRegistry/types/response.ts

```diff
@@ -0,0 +1,32 @@
+import { Attribute, VersionAttribute } from './../../../types/attribute';
+
+export type VersionedAttributeQueryResponse = {
```

Comment:
Why `VersionedAttributeQueryResponse` does contains `VersionedAttribute[]`? Makes it difficult to understand and to change.